### ISSUE TITLE
NPE during compilation with ultra plugin in profiles

### ISSUE THREAD

**Issue by martinhynar** [venantius/ultra#14]
I added ultra 0.1.9 plugin into `profiles.clj` and started to get NPE. Then removed it and compilation went without a glitch. 

```
WARNING: protocol? already refers to: #'clojure.core/protocol? in namespace: potemkin.types, being replaced by: #'potemkin.types/protocol?
java.lang.NullPointerException, compiling:(potemkin.clj:1:1)
```

**Comment by venantius** [comment]
I believe this has been resolved in the latest release, 0.2.0. Please upgrade and let me know if the problem persists.

**Comment by martinhynar** [comment]
Right, 0.2.0 solves it. I haven't noticed that 0.2.0 was released.
